keyboards/tada68/keymaps/devinceble/rules.mk

```diff
@@ -0,0 +1,7 @@
+MOUSEKEY_ENABLE = yes
```

Comment:
This file can be reduced down to just `MOUSEKEY_ENABLE = yes`, the rest are the same as the keyboard-level rules.mk.

Reply:
Noted